update-yarn/create-pr.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -e
+
+# The new Yarn version is passed as an argument to this script
+NEW_YARN_VERSION=$1
+GITHUB_TOKEN=$2
+GITHUB_REPOSITORY=$3
+BRANCH_NAME="yarn-update-$NEW_YARN_VERSION"
+
+# Configure git
+git config --global user.name 'github-actions'
+git config --global user.email 'github-actions@github.com'
+
+# Create a new branch for the Yarn update
+git checkout -b "$BRANCH_NAME"
+
+# Commit and push changes
+git add package.json
+git commit -m "chore: update Yarn to $NEW_YARN_VERSION"
+git push -u origin "$BRANCH_NAME"
+
+# Create a pull request using the GitHub API
+curl -s -X POST -H "Authorization: token $GITHUB_TOKEN" -H "Accept: application/vnd.github.v3+json" \
+  "https://api.github.com/repos/$GITHUB_REPOSITORY/pulls" \
+  -d "{\"title\":\"chore: update Yarn to $NEW_YARN_VERSION\",\"head\":\"$BRANCH_NAME\",\"base\":\"main\",\"body\":\"This PR updates Yarn to the latest release $NEW_YARN_VERSION.\"}"
```

Comment:
Consider using the `gh` command line utility for this (unless you have a very specific reason for using CURL)

See housekeeping script for [example](https://github.com/equinor/fusion-cpr-api/blob/main/.github/scripts/create_housekeeping_pr.sh)

Reply:
Good idea 👍